assets/stylesheets/site/components/_variables.scss

```diff
@@ -106,7 +106,7 @@ $input-max-width:     46rem !default;
 $border-radius:       rem(3px) !default;
 $box-shadow:          0 0 2px $color-shadow !default;
 $focus-shadow:        0 0 3px $color-focus, 0 0 7px $color-focus !default;
-$nav-width:           951px !default;
+$nav-width:           64em !default;
```

Comment:
Changing the `$nav-width` to match USDS allows all the adjustments to occur at the same place.